### DISCUSSION TITLE
满足纳入cdnjs加速列表的要求

### DISCUSSION THREAD

**Discussion by liziwl** [walinejs/waline#756] [Q&A] [answered]
这里申请：https://github.com/cdnjs/packages/issues/new/choose

申请要求：Please ensure the library you are requesting has at least 800 downloads per month on NPM or over 200 stars on GitHub / GitLab / Bitbucket.

看起来应该满足纳入要求。加入 cdnjs 之后一些国内的下游cdn，比如 bootcdn，字节跳动静态资源公共库，360 前端静态资源库都能同步上。

**Comment by Mister-Hope** [comment, marked as answer]
Added
